readme.md

```diff
@@ -290,6 +290,12 @@
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval) - Learn to develop your own search engine.
 
 
+## Natural sciences
```

Comment:
If you add a new top-level entry, then it should also be mentioned in the ToC. And I would create a separate PR for that maybe.